### DISCUSSION TITLE
1000 katılımcıdan sonra paylaşılacak 300.000 parça ibxc hakkında

### DISCUSSION THREAD

**Discussion by raintures** [IBAX-io/go-ibax#1221] [Ecolibs]
@arslanaybars
merhaba sevgili yöneticim, en yüksek puana sahip 1000 katılımcıdan sonra paylaşılacak 300.000 parça ibxc hakkında bilgi verebilir misiniz, 1.000 katılımcıdan sonra herkes için mi yoksa sonraki 1.000 katılımcı için mi geçerlidir?

**Comment by raintures** [comment]
good

**Reply by erhanqwerty** [reply]
kalan herkese eşit
